firebase_connector/models/firebase_notification.py

```diff
@@ -66,6 +66,7 @@ class FirebaseNotification(models.Model):
         string="Partner read status of the notification",
         readonly=True,
     )
+    language = fields.Selection("_get_lang")
```

Comment:
Well, `_get_lang` should be a method defined in your class. I think this alone won't work.